lib/postgrex/connection.ex

```diff
@@ -85,16 +85,38 @@ defmodule Postgrex.Connection do
     message = {:query, statement, params}
     timeout = opts[:timeout] || @timeout
     case Connection.call(pid, message, timeout) do
-      %Postgrex.Result{} = res ->
-        {:ok, res}
-      %Postgrex.Error{} = err ->
-        {:error, err}
       {:error, kind, reason, stack} ->
         :erlang.raise(kind, reason, stack)
```

Comment:
Ok so I should just remove this then and just return the result of `Connection.call/3`? How will it confuse, btw @fishcakez?